specification/imagebuilder/resource-manager/Microsoft.VirtualMachineImages/stable/2020-02-14/imagebuilder.json

```diff
@@ -396,6 +396,9 @@
           }
         ],
         "responses": {
+          "200": {
+            "description": "The retrieval of the operation was successful."
```

Comment:
nit: Either "The operation was successful" Or more specific "The Cancel Operation was successful."

Reply:
updated. thx